### DISCUSSION TITLE
Multiple src with different config

### DISCUSSION THREAD

**Discussion by jofftiquez** [quasarframework/quasar#13264] [Ideas / Proposals]
Hello, first of all thanks for the great framework. I'm a big fan! 

Is there a way to create multiple `src-capacitor/pwa/ssr` in a single project via `quasar.config.js`?

I have 2 mobile apps, one for admin, and one for user, both shares a lot of components from each other but not all android/ios plugins from capacitor. Right now I can't find anything from the documentation related to this. I was able to split everything by modifying the config file programmatically, e.g., theme, htmlVariables, envs, etc. But nothing about src-x. Am I missing something? Thanks.

-------------------

Additional info. Here's what I have in mind.

```
src
 |- components
   |- commons
   |- admin
   |- user
 |- assets
   |- commons
   |- admin
   |- user
src-capacitor-admin // generated using config
src-capacitor-user // generated using config
src-pwa-admin // generated using config
src-pwa-user // generated using config
``

**Comment by metalsadman** [comment]
nothing of some sort ootb yet, though you can probably do it with different `.env` settings then using the techniques sited in this section of the docs https://quasar.dev/quasar-cli-vite/handling-process-env#introduction.

**Reply by jofftiquez** [reply]
Thanks @metalsadman I understand. I already thought of this, altho the problem will still persist, for example, what if both app requires different native configurations? Say, the other one needs a camera permission, the other doesn't. Is there a way to change native config based on `.env` values?
